src/components/navbar.tsx

```diff
@@ -9,34 +8,13 @@ const pages = [
   { name: "Account", href: "/account" },
 ];
 
-const Navbar: React.FunctionComponent = () => {
-  const pathname = usePathname();
-
+const Navbar = () => {
   return (
     <>
       <nav className="justify-content group flex flex-col items-start gap-4  sm:flex-row sm:gap-16">
-        <Link
-          href="#"
-          className={pathname == "/" ? "text-primary" : "primary-dark"}
-        ></Link>
-        <Link
-          href="#"
-          className={pathname == "/" ? "text-primary" : "primary-dark"}
-        >
-          Catalog
-        </Link>
-        <Link
-          href="#"
-          className={pathname == "/manage" ? "text-primary" : "primary-dark"}
-        >
-          Manage
-        </Link>
-        <Link
-          href="#"
-          className={pathname == "/account" ? "text-primary" : "primary-dark"}
-        >
-          Account
-        </Link>
+        {pages.map((page) => (
+          <NavLink href={page.href}>{page.name}</NavLink>
```

Comment:
In this cases it is reccomended to include a key prop to identify the item
For Example:
`<NavLink href={page.href} key={page.name}>{page.name}</NavLink>`